src/Mvc/Mvc.Core/src/MvcOptions.cs

```diff
@@ -337,16 +337,21 @@ public int MaxModelBindingRecursionDepth
         }
 
         /// <summary>
-        /// Gets the <see cref="JsonSerializerOptions"/> used by <see cref="SystemTextJsonInputFormatter"/> and
+        /// Gets the <see cref="System.Text.Json.Serialization.JsonSerializerOptions"/> used by <see cref="SystemTextJsonInputFormatter"/> and
         /// <see cref="SystemTextJsonOutputFormatter"/>.
         /// </summary>
-        public JsonSerializerOptions SerializerOptions { get; } = new JsonSerializerOptions
+        public JsonSerializerOptions JsonSerializerOptions { get; } = new JsonSerializerOptions
```

Comment:
@pranavkm - this seemed right to change. It felt right at the time to make it `SerializerSettings` but I think this is better. when a user looks for this they are thinking "JSON" not "Serializer".

Thoguhts?